### ISSUE TITLE
displaylink driver fails to start in R116 stable

### ISSUE THREAD

**Issue by weberk** [sebanc/brunch#1936]
Just updated from R113 to R116  and realized the following in dmesg:
```
[12330.983240] usb 2-1.1: Product: ThinkPad USB 3.0 Ultra Dock                                                          
[12330.983241] usb 2-1.1: Manufacturer: DisplayLink                                                                     
[12330.983243] usb 2-1.1: SerialNumber: 4551152301011272M2VK                                                            
[12331.040180] usbcore: registered new interface driver cdc_ether                                                       
[12331.068259] init: displaylink-driver pre-start process (5385) terminated with status 1 
```
External display in fact remains dark. Hints appreciated for workarround.

**Comment by ZProLegend007** [comment]
Try the latest brunch-unstable build. It may contain a fix. https://github.com/sebanc/brunch-unstable/releases/tag/r117-unstable-20231007

**Comment by weberk** [comment]
Fantastic! It is working again in [r117-unstable-20231007]! Thanks for pointing out or fixing.

**Comment by weberk** [comment]
Fixed in [r117-unstable-20231007](https://github.com/sebanc/brunch-unstable/releases/tag/r117-unstable-20231007)
